active-elastic-job.gemspec

```diff
@@ -20,19 +20,21 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.5'
+  spec.required_ruby_version = '>= 2.7'
 
   spec.add_dependency 'aws-sdk-sqs', '~> 1'
-  spec.add_dependency 'rails', '>= 5.0', '<7'
+  spec.add_dependency 'rails', '>= 5.2.6', '<= 7.0.1'
```

Comment:
I think you meant `< 7.1` ?

Reply:
I didn't, but you're right that would make more sense as `< 7.1` I'll update.